types/src/network_address/encrypted.rs

```diff
@@ -29,7 +17,7 @@ pub const KEY_LEN: usize = 32;
 pub type Key = [u8; KEY_LEN];
 pub type KeyVersion = u32;
 
-/// Constant key + version so we can push `EncNetworkAddress` everywhere
+/// Constant key + version so we can push `NetworkAddress` everywhere
```

Comment:
Looks good, you should be able to drop the rest of this file now (as it shouldn't be used anywhere else).

Reply:
@gregnazario I think `TEST_SHARED_VAL_NETADDR_KEY` and `TEST_SHARED_VAL_NETADDR_KEY_VERSION` are still being used no? ie. in `config > management > network-address-encryption > src > lib.rs`

Reply:
Yeah, you can remove them.  They're only used to decrypt the encrypted addresses you just removed.  It's an iterative process just try not to dig yourself too far down.